src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -335,7 +338,7 @@ AthensSurfaceExamples class >> example5 [
 			])
 	].
 
-	Display getCanvas drawImage: 	surf asForm  at: 0@0
+	surf asForm asMorph openInWindow. 
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?